README.md

```diff
@@ -19,6 +19,49 @@ ROCm supports programming models, such as OpenMP and OpenCL, and includes all ne
 source software compilers, debuggers, and libraries. ROCm is fully integrated into machine learning
 (ML) frameworks, such as PyTorch and TensorFlow.
 
+## Getting the ROCm Source Code
+
+AMD ROCm is built from open source software. It is, therefore, possible to modify the various components of ROCm by downloading the source code and rebuilding the components. The source code for ROCm components can be cloned from each of the GitHub repositories using git.  For easy access to download the correct versions of each of these tools, the ROCm repository contains a repo manifest file called default.xml. You can use this manifest file to download the source code for ROCm software.
```

Comment:
Nicely written, Randy. Do we want to add a link to the repo with default.xml?

Reply:
Thanks Roopa. I don't think a link is necessary as the default.xml is in the same level of the ROCm repo as the README file.

Reply:
Thanks for confirming, Randy. Sounds good.

Reply:
Thanks for looking.

Reply:
Hi Roopa, I went ahead and added the link while I was editing other things.

Reply:
Excellent! Thanks, Randy.